package.json

```diff
@@ -113,7 +113,7 @@
   },
   "config": {
     "commitizen": {
-      "path": "./app/node_modules/cz-conventional-changelog"
```

Comment:
removed /app to allow us to commit using commitizen

Reply:
thanks!